src/ResultsActivity/elements/PayOptions/PayOptions.js

```diff
@@ -47,35 +47,49 @@ function PayOptions({ offenderDetails, updateOffenderAttendance, date, openModal
     )
 
   const allowUpdate = () => {
-    if (selectedOption === 'other' && isWithinLastYear(date)) return true
+    if (selectedOption === 'other' && isWithinLastWeek(date)) return true
     return false
   }
+  const showRadioButton = () => {
+    if (!locked && offenderNo && eventId && isWithinLastWeek(date)) return true
+    return false
+  }
+
+  const payMsg = () => {
+    const msg =
+      isWithinLastWeek(date) && paid && locked ? 'Paid' : !isWithinLastWeek(date) && !paid && locked && 'Not Paid'
+    return msg
+  }
+
+  const otherMsg = () => {
+    const msg = (other || (!isWithinLastWeek(date) && !paid && !other)) && 'View/ Update'
+    return msg
+  }
 
   return (
     <Fragment>
-      <Option data-qa="pay-option">
+      <Option data-qa="pay-option" className="row-gutters">
+        <PayMessage data-qa="pay-message">{payMsg()}</PayMessage>
         {!isPaying &&
-          !locked &&
-          offenderNo &&
-          eventId && (
+          showRadioButton() && (
             <Radio onChange={payOffender} name={offenderNo} value="pay" checked={selectedOption === 'pay'}>
               <VisuallyHidden>Pay</VisuallyHidden>
             </Radio>
           )}
         {isPaying && <Spinner title="Paying" height={radioSize} width={radioSize} />}
       </Option>
 
-      <Option data-qa="other-option">
-        {!locked &&
-          offenderNo &&
-          eventId && (
-            <>
-              <Radio name={offenderNo} onChange={renderForm} value="other" checked={selectedOption === 'other'}>
-                <VisuallyHidden>Other</VisuallyHidden>
-              </Radio>
-              {allowUpdate() && <UpdateLink onClick={renderForm}>Update</UpdateLink>}
-            </>
-          )}
+      <Option data-qa="other-option" className="row-gutters">
+        {showRadioButton() && (
+          <Radio name={offenderNo} onChange={renderForm} value="other" checked={selectedOption === 'other'}>
+            <VisuallyHidden>Other</VisuallyHidden>
+          </Radio>
+        )}
+        {allowUpdate() && (
+          <UpdateLink onClick={renderForm}>
+            {<OtherMessage data-qa="other-message">{otherMsg()}</OtherMessage>}
```

Comment:
Shouldn't need the `{ }`

Reply:
Also, probably don't want to show `<OtherMessage />` if there is no other message.

Reply:
Added a check in for the other message.  Probably an edge case but it's covered now. (y)